pkgs/by-name/vt/vtsls/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, importNpmLock
+}:
+
+buildNpmPackage rec {
+  pname = "vtsls";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "yioneko";
+    repo = "vtsls";
+    rev = "server-v${version}";
+    hash = "sha256-rHiH42WpKR1nZjsW+Q4pit1aLbNIKxpYSy7sjPS0WGc=";
+    fetchSubmodules = true;
+  };
+
+  sourceRoot = "${src.name}/packages/server";
+
+  npmDeps = importNpmLock {
```

Comment:
I think this is causing IFD. Please see https://github.com/NixOS/nixpkgs/blob/ebc8e4d6d449f84364fa8399f9478fc924ec359b/doc/languages-frameworks/javascript.section.md#tool-specific-instructions-javascript-tool-specific

Reply:
I'm sorry for the delay, I have been flooded with e-mails recently.

@SuperSandro2000 what do you mean with "IFD"? If it's the "import from derivation" feature, I'm not familiar with it and how it could relate to this issue.

I'm not quite sure why ofborg is failing to build it, I can locally via `nix-build -A vtsls` as well as calling the derivation in my system. Reading again the specifications for this, what I'm doing still makes sense to me, maybe I am misinterpreting something.

Reply:
@SuperSandro2000, can you give me an update on this?